streaming/base/shared/prefix.py

```diff
@@ -30,6 +30,19 @@ def _each_prefix_int() -> Iterator[int]:
         prefix_int += 1
 
 
+def _create_filename(prefix_int: int, name: str) -> str:
+    """Get the name of the shared memory.
+
+    Args:
+        prefix (int): The prefix int.
+        name (str): The name of the shared memory.
+
+    Returns:
+        str: Unique shared memory name.
+    """
+    return f'{prefix_int:06}{name}'
```

Comment:
let's put the joining '_' here instead of prefixing the constants with _

Reply:
Done in https://github.com/mosaicml/streaming/pull/299/commits/56368cb07a7657e6e52aef95fb725404c509e6bf.